components/mpas-ocean/src/Registry.xml

```diff
@@ -2670,6 +2702,10 @@
 		<var name="vertGMBolusVelocityTop" type="real" dimensions="nVertLevelsP1 nCells Time" units="m s^{-1}"
 			 description="vertical tracer-transport velocity defined at center (horizontally) and top (vertically) of cell.  This is not the vertical ALE transport, but is Eulerian (fixed-frame) in the vertical, and computed from the continuity equation from the horizontal GM Bolus velocity."
 			 packages="forwardMode;analysisMode"
+		/>
+		<var name="vertMLEBolusVelocityTop" type="real" dimensions="nVertLevelsP1 nCells Time" units="m s^-1"
+			 description="vertical tracer-transport velocity defined at center (horizontally) and top (vertically) of cell.  This is not the vertical ALE transport, but is Eulerian (fixed-frame) in the vertical, and computed from the continuity equation from the horizontal MLE (submesoscale) Bolus velocity."
+			 packages="forwardMode;analysisMode"
```

Comment:
@vanroekel, you're adding several more 3D fields not associated with a package (other than `forwardMode`).  Are these needed even when the new parameterization is turned off?  If so, I guess the new fields are necessary.  If not, it seems like we should create a package for them and not allocate them when `config_submesoscale_enable = .false.`.